src/public/publicAPIs.ts

```diff
@@ -312,4 +312,21 @@ export function navigateToTab(tabInstance: TabInstance, onComplete?: (status: bo
 
   const errorMessage = "Invalid internalTabInstanceId and/or channelId were/was provided";
   GlobalVars.callbacks[messageId] = onComplete ? onComplete : getGenericOnCompleteHandler(errorMessage);
-}
\ No newline at end of file
+}
+
+/**
```

Comment:
I think it might be best if we marked this as @private for now so that it doesn't end up in our documentation until we are ready to ship the feature.